app/src/main/java/woowacourse/payments/ui/component/BankSelectRow.kt

```diff
@@ -0,0 +1,51 @@
+package woowacourse.payments.ui.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.FlowRow
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import woowacourse.payments.domain.model.BankType
+
+private const val COLUMN_COUNT = 4
+
+@Composable
+fun BankSelectRow(
+    onSelect: (BankType) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val banks = remember { BankType.entries.filter { it != BankType.NOT_SELECTED } }
+
+    FlowRow(
+        modifier =
+            modifier
+                .fillMaxWidth()
+                .padding(4.dp)
+                .background(color = Color.White),
+        horizontalArrangement = Arrangement.Center,
+        maxItemsInEachRow = COLUMN_COUNT,
+    ) {
+        banks.forEach { bank ->
+            BankItem(
+                bank = bank,
+                onClick = { onSelect(bank) },
+                modifier = Modifier.width(96.dp),
```

Comment:
```suggestion
                modifier = Modifier.weight(1f),,
```
width(96.dp) 대신 weight(1f)를 사용하면 화면 크기에 맞춰 버튼이 균등하게 분배될 수 있을 거 같아요😊

Reply:
오 이렇게 할 수도 있군요 👍
감사합니다!

[수정했습니당](https://github.com/woowacourse/android-payments/pull/60/commits/92123536b03d009d7e5e53eabfcedf2afaec1b44)